core/src/main/java/nl/nn/adapterframework/configuration/ScheduleManager.java

```diff
@@ -28,14 +28,17 @@
 import lombok.Getter;
 import lombok.Setter;
 import nl.nn.adapterframework.lifecycle.ConfigurableLifecyleBase;
-import nl.nn.adapterframework.lifecycle.ConfiguringLifecycleProcessor;
 import nl.nn.adapterframework.scheduler.SchedulerHelper;
 import nl.nn.adapterframework.scheduler.job.IJob;
 
 /**
- * Configure/start/stop lifecycles are managed by Spring. See {@link ConfiguringLifecycleProcessor}
- *
+ * Container for jobs that are scheduled for periodic execution.
+ * 
  * @author Niels Meijer
+ *
+ */
+/*
+ * Configure/start/stop lifecycles are managed by Spring. See {@link ConfiguringLifecycleProcessor}
```

Comment:
Deze klasse heeft niets met de frankdoc te maken, dit mag weer terug zoals het was.

Reply:
The solution will be issue https://github.com/ibissource/frank-doc/issues/126.